content/en/legal/security-notice.html

```diff
@@ -18,18 +18,18 @@
             <!-- How to report a vulnerability -->
             <h2>How to report a vulnerability</h2>
             <p>CDS is an advocate of responsible vulnerability disclosure. If you’ve found a vulnerability, we would like to know so we can fix it.</p>
-            <p><strong>You can report a vulnerability by sending an email to <a href="mailto:security+securite@cds-snc.ca">security+securite@cds-snc.ca</a></strong>. You can submit reports anonymously. We do not support PGP-encrypted emails at this time.</p>
+            <div class="button-arrow">
+                <p><strong><a href="https://forms-formulaires.alpha.canada.ca/id/105">Report a vulnerability<svg width="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" focusable="false"><path d="M8 9H0V7H8V0L16 8L8 16V9Z"/></svg></a></strong></p> 
+            </div>
+            
             <h3>In your report:</h3>
```

Comment:
Hi Omar, since the below speaks to certain instructions for submitting content, we were thinking maybe it is better if we place it before the new button. what do you think?

Reply:
Yeah, that would work better! Added the screenshots

<img width="1541" alt="Screen Shot 2022-03-01 at 3 03 57 PM" src="https://user-images.githubusercontent.com/33768337/156240481-0cc9ac84-069a-4de1-9775-f48e55adfea7.png">
<img width="1510" alt="Screen Shot 2022-03-01 at 3 03 44 PM" src="https://user-images.githubusercontent.com/33768337/156240486-1a6c3876-8e92-4a48-9ebe-6b694e619db9.png">